src/Microsoft.Graph.Core/Requests/HttpProvider.cs

```diff
@@ -76,11 +65,18 @@ public HttpProvider(IAuthenticationProvider authenticationProvider, HttpMessageH
             {
                 new RedirectHandler(),
                 new RetryHandler(),
-                new AuthenticationHandler(authenticationProvider)
+                new AuthenticationHandler(null)
             };
 
             GraphClientFactory.DefaultHttpHandler = () => this.httpMessageHandler;
             this.httpClient = GraphClientFactory.Create("v1.0", GraphClientFactory.Global_Cloud, handlers);
+            this.httpClient.SetFeatureFlags(new string[]
+            {
+                CoreConstants.FeatureFlags.RedirectHandler,
```

Comment:
This will become something like FeatureFlags.RedirectHandler | FeatureFlags.AuthHandler | FeatureFlags.RetryHandler once we turn this into an enum.